crates/aptos-rosetta/src/error.rs

```diff
@@ -21,7 +21,7 @@ pub enum ApiError {
     InvalidTransferOperations(Option<&'static str>),
     InvalidSignatureType,
     InvalidMaxGasFees,
-    MaxGasFeeTooLow,
+    MaxGasFeeTooLow(Option<String>),
```

Comment:
I wonder why we include the string here if we don't use it below?

Reply:
This is really annoying, but you can't have the "details" in the message by default when listing out the messages, it's a spec thing